dump/plugins/vlan_member.py

```diff
@@ -0,0 +1,144 @@
+from .executor import Executor
+from dump.match_infra import MatchEngine, MatchRequest, ConnectionPool
+from dump.helper import create_template_dict
+
+class Vlan_Member(Executor):
+
+    ARG_NAME = "vlan_member_name"
+    
+    def __init__(self, match_engine=None):
+        super().__init__(match_engine)
+        self.ret_temp = {}
+        self.ns = ''
+          
+    def get_all_args(self, ns=""):
+        req = MatchRequest(db="CONFIG_DB", table="VLAN_MEMBER", key_pattern="*", ns=ns)
+        ret = self.match_engine.fetch(req)
+        all_vlans = ret["keys"]
+        return [(key.split("|")[-2]+'|'+key.split("|")[-1]) for key in all_vlans]
```

Comment:
`[key.split("|", 1)[-1] for key in all_vlans]` Can use something like this, instead of calling the split twice

Reply:
Fixed.